test/modules/carto/mock-fetch.js

```diff
@@ -0,0 +1,15 @@
+/* global global, window */
+
+export function mockTileJSON() {
+  const _global = typeof global !== 'undefined' ? global : window;
+  _global.fetch = url =>
```

Comment:
You need to intercept `fetch` and restore after each test, or this will break other layers.

Reply:
On it, thanks

Reply:
Fixed and I've finally fixed my local dev, I can run browser tests there 🥳